client/homebrew/brewRenderer/brewRenderer.jsx

```diff
@@ -180,7 +212,23 @@ const BrewRenderer = (props)=>{
 			pageText += `\n\n&nbsp;\n\\column\n&nbsp;`; //Artificial column break at page end to emulate column-fill:auto (until `wide` is used, when column-fill:balance will reappear)
 			const html = Markdown.render(pageText, index);
 
-			return <BrewPage className='page' index={index} key={index} contents={html} style={styles} onVisibilityChange={handlePageVisibilityChange} />;
+			// I assume there is a prettier way to do this.
+			const declaredStyles = {};
+			if(brewTemplates[index]?.attr?.styles) {
+				for (let styleSplit of brewTemplates[index]?.attr?.styles.split(';')) {
+					const styleInst = styleSplit.split(':');
+					if(styleInst[0] && styleInst[1]) declaredStyles[styleInst[0].trim()] = styleInst[1].trim();
+				}
+			}
+
+			const styles = {
+				...declaredStyles,
+				...(!displayOptions.pageShadows ? { boxShadow: 'none' } : {}),
+				// Add more conditions as needed
+			};
```

Comment:
Suggested by the toaster, looks cleaner. 

```suggestion
			// I assume there is a prettier way to do this.
const styles = (() => {
    const rawStyles = brewTemplates[index]?.attr?.styles || "";
    const declaredStyles = Object.fromEntries(
        rawStyles.split(";").map(style => {
            const [key, value] = style.split(":").map(s => s?.trim());
            return key && value ? [key, value] : [];
        }).filter(entry => entry.length)
    );

    return {
        ...declaredStyles,
        ...(displayOptions.pageShadows ? {} : { boxShadow: "none" }),
        // Add more conditions as needed
    };
})();

```

Reply:
But use tabs.